packages/adelus/src/CMakeLists.txt

```diff
@@ -0,0 +1,185 @@
+
+INCLUDE(CheckIncludeFileCXX)
+
+#
+# A) Package-specific configuration options
+#
+
+CHECK_INCLUDE_FILE_CXX(sys/time.h HAVE_SYS_TIME_H)
+CHECK_INCLUDE_FILE_CXX(time.h HAVE_TIME_H)
+
+TRIBITS_CONFIGURE_FILE(${PACKAGE_NAME}_config.h)
+
+#
+# B) Define the header and source files (and directories)
+#
+
+#
+# src
+#
+
+SET(HEADERS "")
+SET(SOURCES "")
+
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+
+SET(HEADERS ${HEADERS}
+  ${CMAKE_CURRENT_BINARY_DIR}/${PACKAGE_NAME}_config.h
+  )
+
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Duplicate of line 24?

Reply:
I just followed what Belos and Kokkos Kernels do.